config/detekt.yml

```diff
@@ -106,17 +106,19 @@ formatting:
   active: true
   android: false
   autoCorrect: true
+  ArgumentListWrapping:
+    active: false
   AnnotationOnSeparateLine:
     active: false
     autoCorrect: true
   ChainWrapping:
-    active: true
+    active: false
```

Comment:
Why disabled?

Reply:
And I think this is not consistent with the official style guide
https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping

Reply:
It was conflicting with IDE settings. but will re-enable

Reply:
Oh, that was not my intention to revert your change, rather discuss why did you do so :)